examples/teleportation.ipynb

```diff
@@ -0,0 +1,513 @@
+{
```

Comment:
<p>Could we also show the circuit here in figure?</p><p> </p><p>We can also state that with this teleportation circuit we want to demonstrate how to use measurement and classical controlled gates in <code>QubitCircuit</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>